multiuser/keycloak/che-multiuser-keycloak-server/src/test/java/org/eclipse/che/multiuser/keycloak/server/KeycloakUserManagerTest.java

```diff
@@ -46,6 +46,7 @@
 
   @Mock private UserDao userDao;
   @Mock private ProfileDao profileDao;
+  @Mock private KeycloakProfileRetriever keycloakProfileRetriever;
```

Comment:
Unused import ?

Reply:
Yes. Was introduced by a previous change that I finally reverted. And then I forgot removing it.

Reply:
Fixed in commit 27ad792